### PR TITLE
DLS-3977: Removing google analytics

### PR DESCRIPTION
**Pull request by LGoldfinch2**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -86,7 +86,7 @@ libraryDependencies ++= Seq(
   ws,
   "uk.gov.hmrc"               %% "bootstrap-frontend-play-26"     % "2.24.0",
   "uk.gov.hmrc"               %% "domain"                         % "5.9.0-play-26",
-  "uk.gov.hmrc"               %% "govuk-template"                 % "5.55.0-play-26",
+  "uk.gov.hmrc"               %% "govuk-template"                 % "5.63.0-play-26",
   "uk.gov.hmrc"               %% "play-ui"                        % "8.15.0-play-26",
```
Comment (rambabu-posa) [inline build.sbt:90]: Please remove the below snippet from applicaion.conf

`  google-analytics {
    token = UA-43414424-32
    host = auto
  }`

Comment (LGoldfinch2) [reply build.sbt:90]: My mistake, thanks for the heads up.

File: app/views/govuk_wrapper.scala.html [hunk 2]
```diff
@@ -39,20 +39,6 @@
...
-            ga('create', '@Some(appConfig.analyticsToken)', 'auto');
-
-
-
-            ga('send', 'pageview', { 'anonymizeIp': true });
-    </script>
-    <script src='@routes.Assets.at("javascripts/tabs.js")'></script>
 }
 
 @linksElement = {
```
Comment (rambabu-posa) [inline app/views/govuk_wrapper.scala.html:44]: As per reference PRs we can remove this one also

`@footerLinks = {}`

Comment (LGoldfinch2) [reply app/views/govuk_wrapper.scala.html:44]: Sure, taken that out now.

**Review by rambabu-posa — APPROVED** [review]
LGTM

**Pull request merged**